.github/workflows/build.yml

```diff
@@ -54,12 +54,17 @@ jobs:
   lib-go:
     needs: compiler
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go:
+          - '1.18'
+          - '1.19'
```

Comment:
do you want to only do upload for precross for 1.19?

Reply:
probably, but I'm not sure how to do that with github actions 😅